recipe/conda_build_config.yaml

```diff
@@ -7,6 +7,7 @@ major_minor_ver:
   - 11.1
   - 11.2
   - 11.2.1
```

Comment:
Do we need to keep supporting `11.2` and `11.2.1`? Isn't `11.2.2` enough?

Reply:
AFAIK, only 12.2.2 will be used in the builds.

Reply:
Yeah this only affects the package builds in this feedstocks

Reply:
Sure, but what's the point. We'll have to remember to remove them if we bump the build number in a new PR. So, I think we should just remove building them

Reply:
Why do we have to remember to remove them? If the package has a systematic bug (as has come up with EULA statement script before), we would want to fix older packages as well

Reply:
Ok

Reply:
On second thought, even if the build number is not bumped, the hash might change, leading to new packages being built which is a waste.

Reply:
If that were the case, it would imply we should have bumped the build number, but didn't

In any event if we look at Anaconda.org, we can see there are only 2 packages uploaded recently for Linux (both 11.2.2). Windows hasn't uploaded yet due to the issue noted below ( https://github.com/conda-forge/cudatoolkit-feedstock/pull/57#issuecomment-811311604 ), but should be uploaded soon as well. Other versions haven't seen an upload in over a month

<img width="1792" alt="Screen Shot 2021-03-31 at 11 33 00 AM" src="https://user-images.githubusercontent.com/3019665/113193492-dd049b80-9214-11eb-9e78-be2accccd6d4.png">

Reply:
> Other versions haven't seen an upload in over a month

That's because the gcc compiler version hasn't changed in over a month.

Reply:
If we are concerned about this, we could add the compilers to `ignore_version`

https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#special-variant-keys
https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#avoiding-unnecessary-builds